deploy/deploy.sh

```diff
@@ -44,14 +44,26 @@ echo "[==] Adding tar files"
 mkdir -p ${golden_tar_path}
 for arch in ${unix_archs}; do
   cp husarnet-${package_version}-${arch}.tar ${golden_tar_path}
-  ln -fs husarnet-${package_version}-${arch}.tar ${golden_tar_path}/husarnet-latest-${arch}.tar
 done
 
 echo "[==] Adding pacman files"
 mkdir -p ${golden_pacman_path}
 for arch in ${unix_archs}; do
-  cp husarnet-${package_version}-${arch}.pacman ${golden_pacman_path}
-  ln -fs husarnet-${package_version}-${arch}.pacman ${golden_pacman_path}/husarnet-latest-${arch}.pacman
+
+  mkdir -p ${golden_pacman_path}/${arch}
+  cp husarnet-${package_version}-${arch}.pacman ${golden_pacman_path}/${arch}/husarnet-${package_version}-${arch}.pkg
+  ln -fs ${golden_pacman_path}/${arch}/husarnet-${package_version}-${arch}.pkg ${golden_pacman_path}/${arch}/husarnet-latest-${arch}.pkg
+  if [[ ${arch} -eq armhf ]]; then
+    archlinux_arch_name="armhf"
+  elsif [[ ${arch} -eq arm64 ]]; then
+    archlinux_arch_name="aarch64"
+  elsif [[ ${arch} -eq amd64 ]]; then
+    archlinux_arch_name="x86-64"
+  else
+    archlinux_arch_name=${arch}
+  fi
+  mv ${golden_pacman_path}/${arch} ${golden_pacman_path}/${archlinux_arch_name}
```

Comment:
Why you copy stuff a couple of lines above just to move it here instead of removing those copy lines above and changing mv to cp here and doing the job in one pass?